README.md

```diff
@@ -153,6 +153,21 @@ html lang: "en" do
 end
 ```
 
+Update your config to live-reload the template files.
+```elixir
+# config/dev.exs
+...
+config :your_app, YourAppWeb.Endpoint,
+  live_reload: [
+    patterns: [
+      # ...
+      # replaces ~r"lib/you_app_web/templates/.*(eex)$"
+      ~r"lib/your_app_web/templates/.*(exs)$"
+    ]
+  ]
+...
```

Comment:
Can you add this to the same code block as the other configuration, without the `...` and the description (I think it's obvious what it's doing)